lib/rbac/roles.rb

```diff
@@ -39,6 +39,15 @@ def delete(role)
       end
     end
 
+    def self.assigned_role?(role_name)
+      opts = { :limit => 100,
+               :scope => 'principal' }
+
+      RBAC::Service.call(RBACApiClient::RoleApi) do |api_instance|
```

Comment:
@lindgrenj6  we should add a spec for this method model around this file
https://github.com/ManageIQ/catalog-api/blob/7ae8422696a4cdec369c9e4746a7683f61d4ec6f/spec/lib/rbac/query_shared_resource_spec.rb#L1

We never added a roles spec since that is rolled into other specs.

Reply:
Added a spec for this method.